### PR TITLE
[SRVKS-780] Add Knative Scaling debugging dashboard

### PR DESCRIPTION
**Pull request by skonto**
This is an adaption from https://github.com/knative-sandbox/monitoring/blob/main/grafana/knative-scaling.json. Target is 1.18.

Screenshots follow:

image
Using to test:
```
apiVersion: serving.knative.dev/v1
kind: Service
metadata:
  name: autoscale-go
  namespace: default
spec:
  template:
    metadata:
      annotations:
        # Target 10 in-flight-requests per pod.
        autoscaling.knative.dev/target: "5"
        autoscaling.knative.dev/minScale: "1"
    spec:
      containers:
      - image: gcr.io/knative-samples/autoscale-go:0.1
      
hey  -z 120s -c 100 "http://autoscale-go-default.apps.....?sleep=100&prime=100000&bloat=100"
```

image
Pods number stabilizes:
image
image
image
image

If an rps based service is used:
```
apiVersion: serving.knative.dev/v1
kind: Service
metadata:
  name: helloworld-go
  namespace: default
spec:
  template:
    metadata:
      annotations:
        autoscaling.knative.dev/target: "10"
        autoscaling.knative.dev/metric: "rps"
    spec:
      containers:
        - image: docker.io/skonto/helloworld-go
```
RPS diagram is fed with data:
image

No support for HPA for now.
Scrape time depends on: https://github.com/knative/serving/pull/11778
Line overlapping is being fixed here: https://github.com/openshift/console/pull/9725

/cc @markusthoemmes @nak3 @rhuss

### CODE REVIEW

**Comment by skonto** [comment]
@markusthoemmes @nak3 gentle ping.

File: knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-resources.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ data:
       "annotations": {
         "list": []
       },
-      "description": "Knative Eventing - Source CPU and Memory Usage",
+      "description": "Knative Eventing - Source CPU, Memory and Network Usage",
```
Comment (markusthoemmes) [inline knative-operator/deploy/resources/dashboards/eventing/grafana-dash-knative-eventing-resources.yaml:24]: This seems like a separate change to me, a different PR maybe?

File: knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-resources.yaml [hunk 2]
```diff
@@ -35,7 +21,7 @@ data:
       "annotations": {
         "list": []
       },
-      "description": "Knative Serving - Revision CPU and Memory Usage",
+      "description": "Knative Serving - Revision CPU, Memory and Network Usage",
```
Comment (markusthoemmes) [inline knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-resources.yaml:24]: Same.

**Review by markusthoemmes — COMMENTED** [review]
Can't really review the dashboard code. Not sure if somebody else is better suited for this.

File: knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-resources.yaml (hunk 2 above)
Comment (skonto) [reply knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-resources.yaml:24]: It is too small and trivial. I noticed that when I did write the dashboard so I just added it.

**Comment by skonto** [comment]
@markusthoemmes I dont know either, there is no validation tool for native dashboards either. However it does use what other dashboards use so dont expect issues here. One option to review this try it and test how it looks in practice.

**Comment by maschmid** [comment]
OK, I'll try the "One option to review this is to actually try it and test how it looks/behave in practice." route. It will take some time, we need to check this on all supported OCP versions.

**Comment by maschmid** [comment]
I have verified the dashboards seem to work on 4.6.42, 4.7.24, 4.8.9 and 4.9.0-0.nightly-2021-09-06-055314 , using the autoscale-go and helloworld-go-with-rps ksvcs.

**Review by maschmid — APPROVED** [review]

**Comment by maschmid** [comment]
( I am not an approver, so feel free to approve, if you agree with the description change... )

**Comment by skonto** [comment]
@markusthoemmes gentle ping for approval.

**Review by markusthoemmes — APPROVED** [review]

**Pull request merged**